bin/detect-community-build.sh

```diff
@@ -2,11 +2,18 @@
 
 set -e
 
+MONITOR_REPO="scalacenter/bloop"
 PR="$DRONE_PULL_REQUEST"
+
 if [[ "$PR" ]]; then
-  MONITOR_REPO="scalacenter/bloop"
-  if curl --silent https://api.github.com/repos/"$MONITOR_REPO"/issues/"$PR"/labels | jq -e '.[].name | contains("community-build")'; then
+  if curl --silent https://api.github.com/repos/"$MONITOR_REPO"/issues/"$PR"/labels | jq -e '.[].name | contains("community-build")' > /dev/null 2>&1 ; then
     echo "The community build will be run for this pull request."
-    export RUN_COMMUNITY_BUILD=true
+    exit 0
+  else
+    echo "The community build will not run for this pull request."
+    exit 1
```

Comment:
Doesn't this mean that Drone will fail altogether?

Reply:
Good catch, the problem was actually in `.drone.yml`